features/networking/pod.feature

```diff
@@ -103,23 +103,23 @@ Feature: Pod related networking scenarios
     And evaluation of `pod.node_name` is stored in the :node_name clipboard
 
     # check the ovs port and interface for the qos availibility
-    When I run command on the "<%= cb.node_name %>" node's sdn pod:
-      | ovs-vsctl | list | qos |
+    Given I use the "<%= cb.node_name %>" node
+    And I run commands on the host:
+      | ovs-vsctl list qos |
     Then the step should succeed
     And the output should contain "max-rate="5000000""
-    When I run command on the "<%= cb.node_name %>" node's sdn pod:
-      | ovs-vsctl | list | interface |
+    And I run commands on the host:
+      | ovs-vsctl list interface |
```

Comment:
Agree. I guess we should stick to sdn pod wherever possible.

Reply:
yes, that's a good question.
I'm changing this because:
 When I was debugging this issue in my local. Found when running this step 
```
When I run command on the "<%= cb.node_name %>" node's sdn pod:
      | ovs-vsctl | list | interface |
```
and `ovs-vsctl list interface` always show nothing.  However it works well if run this command on the cluster worker directly.  I 'm thinking if this step `When I run command on the "<%= cb.node_name %>" node's sdn pod` need to find the correct sdn/ovn pod on the worker and print a lot of output.  I guess but not sure if the output consumed a lot of memory and then it affect the output of `ovs-vsctl list interface` .  see logs http://pastebin.test.redhat.com/1021678
But works well when updating to `And I run commands on the host`

Reply:
@zzhao you can ```quiet: true``` after ```user: admin``` to supress the logs here https://github.com/openshift/verification-tests/blob/master/features/step_definitions/networking.rb#L645 and https://github.com/openshift/verification-tests/blob/master/features/step_definitions/networking.rb#L651. WDYT

Reply:
yes, @anuragthehatter you are correct. After adding `quiet: true`,  it works now. 
and also change others for same reason.

Reply:
Awesome. Thanks @zhaozhanqi for fixing others as well.